README.md

```diff
@@ -148,6 +148,21 @@ custom:
             - "$context.identity.sub"
             - "$context.arguments.id"
           ttl: 1000 # override the ttl for this resolver. (default comes from global config)
+        # When versioning is enabled with `versioned` on the datasource,
+        # the datasync options of the resolver.
+        # Disabled by default.
+        # Accepted values:
+        # - `true`: sync enabled with default ConflictDetection VERSION
+        # - an object with the following keys:
+        #    - ConflictDetection: The Conflict Detection strategy to use.
+        #    - lambda: true or false, when specified pass in the lambdaArn
+        #    - lambdaArn: The Arn for the Lambda function to use as the Conflict Handler.
+        #    - ConflictHandler: when lambda is false use the Conflict Resolution strategy to perform in the event of a conflict.
```

Comment:
@hounded Let's try to keep naming convention and use camelCase for all config properties.

Reply:
sorry copy pasted in the ConflictDetection ConflictHandler from the aws json doc